### DISCUSSION TITLE
Wi-Fi automatically connecting when not isOnline

### DISCUSSION THREAD

**Discussion by sethhammett** [koreader/koreader#7692] [Q&A] [answered]
Sorry to bother you again but I have another Wi-Fi problem and this time it seems to be an issue with the code.
In .adds/koreader/frontend/ui/network/manager.lua in line 282 it looks like if the device is connected to a network but can't reach a server it will automatically connect to the network again. I tried to connect to a Wi-Fi hotspot earlier before I realized that KOReader doesn't have a browser so now I seem to be stuck in an infinite doomloop of the device trying to connect to the same network, even when I press and hold on the menu option it shows the networks but the connects to the broken one. I deleted .adds/koreader/settings/network.lua and restarted KOReader but nothing changed.

**Comment by NiLuJe** [comment, marked as answer]
What you linked to is unrelated.

What seems to be happening is that the hotspot AP is still accessible and either provides better link quality than other favorite/saved APs *OR* you don't actually have any other saved APs.

Provided you don't have "close the AP list on connect" enabled, the AP list should stay up on a long-press, which should allow you to disconnect from the useless AP and connect to another.
